src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawerAdapter.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+public class DrawerAdapter implements Job2dDriver {
+
+    private final DrawerFeature drawerFeature;
+    private final ILine line;
+    private int startX;
+    private int startY;
+
+    public DrawerAdapter() {
+        super();
+        this.drawerFeature = new DrawerFeature();
+        this.line = LineFactory.getBasicLine();
+    }
+
+    public DrawerAdapter(ILine line) {
+        super();
+        this.line = line;
+        this.drawerFeature = new DrawerFeature();
```

Comment:
Direct coupling with DrawerFeature class is unnecessary.
Creating instance of DrawerFeature is also unnecessary.